test/e2e-upgrade-tests.sh

```diff
@@ -74,10 +74,26 @@ header "Running preupgrade tests"
 go_test_e2e -tags=preupgrade -timeout=${TIMEOUT} ./test/upgrade \
   --resolvabledomain=$(use_resolvable_domain) || fail_test
 
+header "Starting prober test"
+
+go_test_e2e -tags=probe -timeout=${TIMEOUT} ./test/upgrade \
+  --resolvabledomain=$(use_resolvable_domain) &
+PROBER_PID=$!
+echo "PROBER_PID=$PROBER_PID"
```

Comment:
```suggestion
echo "Prober PID is ${PROBER_PID}"
```

Reply:
I suggest adding a `rm -f /tmp/prober-signal` here for safety, in case you ran this test multiple times locally.